ui/app/templates/components/mfa/method-list-item.hbs

```diff
@@ -30,30 +30,26 @@
     </div>
     <div class="level-right is-flex is-paddingless is-marginless">
       <div class="level-item">
-        <PopupMenu>
-          <nav class="menu" aria-label="navigation for managing MFA methods">
-            <ul class="menu-list">
-              <li>
-                <LinkTo
-                  @route="vault.cluster.access.mfa.methods.method"
-                  @model={{@model.id}}
-                  data-test-mfa-method-menu-link="details"
-                >
-                  Details
-                </LinkTo>
-              </li>
-              <li>
-                <LinkTo
-                  @route="vault.cluster.access.mfa.methods.method.edit"
-                  @model={{@model.id}}
-                  data-test-mfa-method-menu-link="edit"
-                >
-                  Edit
-                </LinkTo>
-              </li>
-            </ul>
-          </nav>
-        </PopupMenu>
+        <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
+          <dd.ToggleIcon
+            @icon="more-horizontal"
+            @text="Manage M-F-A method"
```

Comment:
Should this be "M-F-A" or "MFA"?

Reply:
I noticed we do this for AWS below, so it seems to be a convention. Feel free to disregard my comment!

Reply:
It's for accessibility purposes!